src/Consolonia.Gallery/View/ControlsListView.axaml

```diff
@@ -63,15 +63,18 @@
             </MenuItem>
         </Menu>
 
-        <DataGrid x:Name="GalleryGrid"
-                  DockPanel.Dock="Left"
-                  SelectedIndex="0"
-                  SelectionMode="Single">
-            <DataGrid.Columns>
-                <DataGridTextColumn Header="Name"
-                                    Binding="{Binding Name}" />
-            </DataGrid.Columns>
-        </DataGrid>
+        <ListBox x:Name="GalleryGrid"
+                 DockPanel.Dock="Left"
+                 SelectedIndex="0"
+                 Focusable="true"
+                 Width="24"
```

Comment:
Why do we need fixed width?

Reply:
![Image](https://github.com/user-attachments/assets/b236657c-a124-4ed2-8d35-2dbcd069b056)